src/App.tsx

```diff
@@ -0,0 +1,53 @@
+import './index.css';
+import './assets/styles/global.css';
+import LandingPage from './view/LandingPage/LandingPage';
+import { BrowserRouter as Router, Route, Switch } from 'react-router-dom';
+import MapView from './view/MapView/MapView';
+import Members from './view/Members/Members';
+import AdminPanel from './admin/adminPanel/AdminPanel';
+import FlightsProvider from './Providers/FlightsProvider/FlightsProvider';
+import PostsProvider from './Providers/PostsProvider/PostsProvider';
+import { FC, MouseEvent } from 'react';
+import { useState, useEffect, useRef } from 'react';
+
+interface ICoords {
```

Comment:
What do you think about keeping this kind of interfaces in some special directory like model, interfaces, utils or smth like that?

Reply:
There's the flights.d.ts file that stores globally used interfaces. I decided to keep the ones used in only one component (or only parent-child) locally.

Reply:
Oki